src/Microsoft.Extensions.Configuration.AzureAppConfiguration/KeyValueChange.cs

```diff
@@ -23,5 +23,7 @@ internal struct KeyValueChange
         public ConfigurationSetting Current { get; set; }
 
         public ConfigurationSetting Previous { get; set; }
+
+        public bool IsMultiKeyWatcherChange { get; set; }
```

Comment:
The name "IsWatchedSettings"  is easier to understand and also the usage will be like.

```
if (change.IsWatchedSettings)
{
    watchedSettings[changeIdentifier] = settingCopy;
}
```

Instead of 
```
if (!change.IsMultiKeyWatcherChange)
{
    watchedSettings[changeIdentifier] = settingCopy;
}
```

Reply:
Yea makes sense to match the name of `watchedSettings` when that's the use here